perdiem.py

```diff
@@ -409,10 +426,10 @@ def find_missing_records(stays, results):
 
 if __name__ == '__main__':
     """Currently uses web-scraping3 python virtual env"""
-    data = 'stays/All_Stays_2019Q2.csv'
+    data = 'stays/All_Stays_2020Q3.csv'
     # Year and quarter for output file naming
-    utah_fiscal_year = '2019'
-    quarter = 'q2'
+    utah_fiscal_year = '2020'
+    quarter = 'q3'
```

Comment:
**suggestion**: Something for the future. These could be made into arguments so that you don't have to make new commits to the code each quarter.